dev/com.ibm.websphere.appserver.features/visibility/public/mail-2.0/resources/l10n/io.openliberty.mail-2.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature allows applications to utilize the Jakarta Mail 2.0 API. 
```

Comment:
suggested edit:

This feature allows applications to use the Jakarta Mail 2.0 API.